common/test/acceptance/tests/lms/test_lms_instructor_dashboard.py

```diff
@@ -1202,3 +1202,107 @@ def test_invalidate_certificates_a11y(self):
             '.certificates-wrapper'
         ])
         self.certificates_section.a11y_audit.check_for_accessibility_errors()
+
+
+@attr(shard=10)
+class EcommerceTest(BaseInstructorDashboardTest):
+    """
+    Bok Choy tests for the "E-Commerce" tab.
+    """
+    def setUp(self):
+        super(EcommerceTest, self).setUp()
+
+    def setup_course(self, course_number):
+        """
+        Sets up the course
+        """
+        self.course_info['number'] = course_number
+        course_fixture = CourseFixture(
+            self.course_info["org"],
+            self.course_info["number"],
+            self.course_info["run"],
+            self.course_info["display_name"]
+        )
+        course_fixture.install()
+
+    def log_in_as_unique_user(self):
+        """
+        Log in as a valid lms user.
+        """
+        AutoAuthPage(
+            self.browser,
+            username="test_instructor",
+            email="test_instructor@example.com",
+            password="password",
+            course_id=self.course_id
+        ).visit()
+
+    def visit_ecommerce_section(self):
+        """
+        Log in to visit Instructor dashboard and click E-commerce tab
+        """
+        self.log_in_as_unique_user()
+        instructor_dashboard_page = self.visit_instructor_dashboard()
+        return instructor_dashboard_page.select_ecommerce_tab()
+
+    def add_course_mode(self, sku_value=None):
+        """
+        Add an honor mode to the course
+        """
+        ModeCreationPage(self.browser, self.course_id, min_price=10, sku=sku_value).visit()
+
+    def test_enrollment_codes_section_visible_for_non_ecommerce_course(self):
+        """
+        Test Enrollment Codes UI, under E-commerce Tab, should be visible in the Instructor Dashboard with non
+        e-commerce course
+        """
+        # Setup course
+        self.setup_course("34039497242734583224814321005482849780")
```

Comment:
Maybe I don't quite understand -- are we specifying a new course number, or are we referencing an existing course number that we were able to determine?  It seems like we should not have static course number references in the test harness -- we should create a course and then use the number from the newly-created course, if possible.